contrib/harfbuzz/module.defs

```diff
@@ -18,11 +14,7 @@ HARFBUZZ.CONFIGURE.extra = \
     FREETYPE_CFLAGS="-I$(call fn.ABSOLUTE,$(CONTRIB.build/))include/freetype2" \
 
 ifneq ($(BUILD.system),darwin)
```

Comment:
You would need to update the comment at the beginning of that section if you do (I realize we haven't been consistent in that regard in many places, ourselves!).